asynctask/src/main/java/com/example/asynctask/SimpleAsyncTask.kt

```diff
@@ -0,0 +1,60 @@
+package com.example.asynctask
+
+import android.os.AsyncTask
+import android.view.View
+import android.widget.ProgressBar
+import android.widget.TextView
+import java.lang.ref.WeakReference
+import java.util.Random
+/**Params: Là giá trị ((biến) được truyền vào khi gọi thực thi tiến trình và nó sẽ được truyền vào doInBackground
+Progress: Là giá trị (biến) dùng để update giao diện diện lúc tiến trình thực thi, biến này sẽ được truyền vào hàm onProgressUpdate
+Result: Là biến dùng để lưu trữ kết quả trả về sau khi tiến trình thực hiện xong
+Mặc định sẽ là Void
+ **/
```

Comment:
chỉ cần comment giải thích những gì gây khó hiểu như là thuật toán khó hay đặc thù của bài toán phải như vậy , còn những thứ thuộc về cơ bản của công nghệ thì ko cần